### PR TITLE
build(aio): align stackblitz files with Angular CLI V6

### LINKED ISSUE #23437: build(aio): ensure stackblitz examples look like cli@6
Although the examples have been switched to angular@6 and cli@6, the stackblitz projects still contain cli@1 stuff (e.g. `.angular-cli.json`).

The live examples should look like cli@6 projects.

### PR DESCRIPTION
**Pull request by brandonroberts**
The `.angular-cli.json` file was still being copied into the stackblitz examples

Closes #23437

## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[ ] Documentation content changes
[x] angular.io application / infrastructure changes
[ ] Other... Please describe:
```

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?
```
[ ] Yes
[x] No
```

### CODE REVIEW

File: aio/tools/stackblitz-builder/builder.js [hunk 1]
```diff
@@ -252,7 +252,7 @@ class StackblitzBuilder {
     }
 
     var defaultIncludes = ['**/*.ts', '**/*.js', '**/*.css', '**/*.html', '**/*.md', '**/*.json', '**/*.png'];
-    var boilerplateIncludes = ['src/environments/*.*', '.angular-cli.json', 'src/polyfills.ts'];
+    var boilerplateIncludes = ['src/environments/*.*', 'angular.json', 'src/polyfills.ts'];
```
Comment (gkalpak) [inline aio/tools/stackblitz-builder/builder.js]: But why does the [file exist][1] in the first place? Shouldn't it be removed?

[1]: https://github.com/brandonroberts/angular/blob/abd88d9d49186eb6d1564dc2270bb942a27fbf3d/aio/tools/examples/shared/boilerplate/cli/.angular-cli.json

**Review by gkalpak — APPROVED** [review]
Unrelated to this PR, but I noticed the [universal][1] and [testing][2] examples still use the old cli + layout. Is there a reason or is it temporary? If the intention is to eventually upgrade them too, can you please create an issue, @brandonroberts?

[1]: https://github.com/brandonroberts/angular/blob/abd88d9d49186eb6d1564dc2270bb942a27fbf3d/aio/tools/examples/example-boilerplate.js#L60
[2]: https://github.com/brandonroberts/angular/blob/abd88d9d49186eb6d1564dc2270bb942a27fbf3d/aio/tools/examples/example-boilerplate.js#L66

**Comment by brandonroberts** [comment]
@gkalpak the intention is to upgrade those also. They require changes to the associated guides also and the initial PR was to upgrade the standard examples that didn't require any custom configuration. I'll add an issue for updating the remaining ones. Once those are updated, then we can safely remove the `.angular-cli.json` from the shared example files.

**Comment by gkalpak** [comment]
> They require changes to the associated guides also and the initial PR was to upgrade the standard examples that didn't require any custom configuration.

:+1:

> Once those are updated, then we can safely remove the `.angular-cli.json` from the shared example files.

If it is only `universal` and `testing`, then do we still need the [boilerplate/cli/.angular-cli.json][1] (given that [universal][2] and [testing][3] use their own boilerplate `.angular-cli.json`)?

[1]: https://github.com/brandonroberts/angular/blob/abd88d9d49186eb6d1564dc2270bb942a27fbf3d/aio/tools/examples/shared/boilerplate/cli/.angular-cli.json
[2]: https://github.com/brandonroberts/angular/blob/abd88d9d49186eb6d1564dc2270bb942a27fbf3d/aio/tools/examples/shared/boilerplate/universal/.angular-cli.json
[3]: https://github.com/brandonroberts/angular/blob/abd88d9d49186eb6d1564dc2270bb942a27fbf3d/aio/tools/examples/shared/boilerplate/testing/.angular-cli.json

**Comment by brandonroberts** [comment]
@gkalpak I don't think so. I'll remove it as part of this PR and any leftover references.

File: aio/tools/stackblitz-builder/builder.js (hunk 1 above)
Comment (IgorMinar) [reply aio/tools/stackblitz-builder/builder.js]: does stackblitz use this file for anything?

Comment (gkalpak) [reply aio/tools/stackblitz-builder/builder.js]: AFAICT, it only uses it to include it in the exported zip.
(BTW, I just realized the zips exported by StackBlitz are broken (e.g. wrong layout) for both old and new projects.)

**Comment by gkalpak** [comment]
Why didn't Travis pick this up? 😒

**Comment by brandonroberts** [comment]
@gkalpak this one is ready for another review

**Comment by gkalpak** [comment]
Did you make any more changes other than removing `boilerplate/cli/.angular-cli.json`?

(BTW, once a PR has been reviewed (and especially if you aren't making significant changes), it helps not amend the original commit, but add a `fixup` commit instead 💥)

**Review by gkalpak — APPROVED** [review]

**Pull request closed without merging**